### PR TITLE
Update from Intelsdi-x pluginsync 0.1.8

### PR DESCRIPTION
**Pull request by katarzyna-z**
Summary of changes:

- Update from Intelsdi-x pluginsync 0.1.8
- Removed snap-pluginsync control for scripts/build.sh file to preserve custom sections in this file.

### CODE REVIEW

File: scripts/build.sh [hunk 1]
```diff
@@ -46,12 +46,3 @@ export GOOS=linux
 export GOARCH=amd64
 mkdir -p "${build_dir}/${GOOS}/x86_64"
 "${go_build[@]}" -o "${build_dir}/${GOOS}/x86_64/${plugin_name}" . || exit 1
-
```
Comment (nanliu) [inline scripts/build.sh]: We need to preserve this custom section, please configure this file to be ignored in .sync.yml:

```
scripts/build.sh:
  unmanaged: true
```

Comment (katarzyna-z) [inline scripts/build.sh]: I removed changes in scripts/build.sh to preserve this custom section.

**Comment by nanliu** [comment]
Please combine PR #46 with #45. Thanks!

**Comment by katarzyna-z** [comment]
Pull requests #45 and #46 are combined.

**Pull request merged**